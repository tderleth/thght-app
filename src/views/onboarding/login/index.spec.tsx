import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Login from '.';

describe('Login', () => {
  it('snapshot test', () => {
    const tree = renderer.create(<Login />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
