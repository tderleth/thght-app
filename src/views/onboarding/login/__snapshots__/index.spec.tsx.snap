// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login snapshot test 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "color": "white",
      "flex": 1,
      "fontSize": 30,
      "justifyContent": "center",
    }
  }
>
  <Text>
    This will be the login screen.
  </Text>
</View>
`;
